doc/source/dev/contributor/pep8.rst

```diff
@@ -39,6 +39,22 @@ compliance before pushing your code:
    right, consider introducing (and then removing) a small PEP8 issue in
    the target file, such as a line that is over 79 characters long.
 
+-  Before sending a Pull Request, we recommend running the lint tests only
+   for the changes you've made in your feature branch. This will mimic
+   the continuous integration linting checks setup on GitHub.
+   You can run the following in the SciPy root directory.
+
+   ::
+
+      python tools/lint_diff.py
+
+   If you want to run the diff based lint tests only for specific files
+   or directories, please consider using the ``--files`` option.
+
+   ::
+
+      python tools/lint_diff.py --files scipy/odr/models.py scipy/ndimage
```

Comment:
Everything worked for me except multiple files/folders:
```
(scipydev) Matts-MacBook-Pro:scipy matthaberland$ python tools/lint_diff.py --files scipy/stats/tests/test_hypotests.py scipy/stats
```
(no output)
```
(scipydev) Matts-MacBook-Pro:scipy matthaberland$ python tools/lint_diff.py --files scipy/stats/tests/test_hypotests.py
scipy/stats/tests/test_hypotests.py:1170:17: E231 missing whitespace after ','
```

Reply:
Yes, it seems to work now.

> Are you also suggesting adding a print statement even on success?

I was. I know that `flake8` doesn't produce any output; I just disagree with that convention. "Explicit is better than implicit."

Reply:
I agree with the zen of python. Made the change.